docs/user/datastores/index_config.rst

```diff
@@ -649,6 +649,19 @@ For example:
 
 See :ref:`cli_analytic` and :ref:`stats_api` for information on reading cached stats.
 
+Configuring Temporal Priority
+-----------------------------
+
+For some large time-based datasets, an index that leverages a temporal predicate will almost always be faster
```

Comment:
If we believe this, should we make the default 'true'?

Reply:
well, there's the `For some large time-based datasets` part...

Reply:
in practice, i think the only thing this affects is prefix attribute scans against high-cardinality attributes get de-prioritized below z3.